pages/linux/tlp-stat.md

```diff
@@ -0,0 +1,15 @@
+# tlp-stat
+
+> TLP status reports. See also `tlp`.
```

Comment:
How about - "A tool to generate TLP status reports." ?

Reply:
I'm going to recommend we go with what's on the [man page](https://www.mankier.com/8/tlp-stat) if you want more descriptive text. How about the following:

> Show configuration, system information, active power saving settings and battery data.

I'll update my PR with the above. If you'd still prefer your suggested verbiage, let me know. Thanks!

Reply:
Your original description was fine. There is no need to describe everything in the heading. I just wanted to add "A tool to generate" to clarify what it does. Otherwise if you just say "TLP status reports", then it is incomplete, and questions like "Where, What" arise.

Reply:
Makes sense. I will revert and update with your suggestion.

FWIW - as `tlp-stat` installs with `tlp`, and isn't used on it's own, I had made the incorrect assumption that level of specificity wasn't needed.

Thanks for clarification. Will update now.